workers/tests/integrationtests/test_api_indicator.py

```diff
@@ -21,9 +21,9 @@ class TestApiIndicator(unittest.TestCase):
     def setUp(self):
         self.client = TestClient(app)
         self.dataset = "regions"
-        self.feature_id = 31
+        self.feature_id = 1
```

Comment:
Why did you change the feature *only* for this test? What is different from the others using `31` right now?

Reply:
Reverted the change